features/step_definitions/fragment.rb

```diff
@@ -0,0 +1,62 @@
+require 'capybara/cucumber'
+require 'selenium/webdriver'
+
+Capybara.run_server = false
+Capybara.default_driver = :selenium_chrome_headless
+Capybara.app_host = "http://localhost:3000"
+Capybara.default_max_wait_time = 10
+
+# Conditions
+
+Soit("l'item {string} rattaché au corpus {string}") do |string, string2|
+   # On the remote servers
+end
+
+Soit("le point de vue {string} rattaché à l'item {string}") do |string, string2|
+   # On the remote servers
+end
+
+Soit("la catégorie {string} rattaché au point de vue {string}") do |string, string2|
+   # On the remote servers
+end
+
+Soit("le fragment {string} rattaché à la catégorie {string}") do |string, string2|
+   # On the remote servers
+end
+
+Alors("il doit y avoir au moins {int} items affichés") do |int|
+   expect(page).to have_selector('.Items .item', count: int)
+end
+
+Alors("l'item {string} est décrit par une date") do |item|
```

Comment:
A `date`? Which one? 
Please note that scenarios should include samples therefore steps should include parameters.